internal/cli/atlas/security/ldap/save.go

```diff
@@ -65,6 +68,23 @@ func (opts *SaveOpts) Run() error {
 	return opts.Print(r)
 }
 
+func (opts *SaveOpts) Prompt() error {
+	if opts.bindPassword != "" {
+		return nil
+	}
+
+	if !opts.IsTerminalInput() {
+		_, err := fmt.Fscanln(opts.InReader, &opts.bindPassword)
+		return err
+	}
+
+	prompt := &survey.Password{
+		Message: "Password:",
+	}
+
+	return survey.AskOne(prompt, &opts.bindPassword)
+}
+
```

Comment:
[nit] I think we could add this function under `InputOpts` and pass the `password` as an input parameter so that we can reuse this piece of code. Then you can call this function in any commands that require this prompt message

Reply:
@andreaangiolillo - is this a change you'd like me to introduce as part of this PR?

Reply:
It is up to you. We can add this improvement in another PR if you want ; )